### PR TITLE
Gather aois 119

### LINKED ISSUE #119: gather AOIs used in repo
gather all the AOIs used in the notebooks repo into one geojson file

### PR DESCRIPTION
**Pull request by jreiberkyle**
Gather the aois used in this repository into a single, easy to navigate geojson file and provide a notebook for maintaining the aoi feature, along with guidelines in the CONTRIBUTING doc.

Blocked by #118 
Closes #119

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,7 +11,23 @@ because pytest does not like spaces in command line arguments.
...
+already used in this repository. The geojson description of these AOIs is given
+in [aois.geojson](dev/imports_test.ipynb) (it is easy to visualize these
+aois directly in GitHub or by copy/pasting into [geojson.io](geojson.io).
+These AOIs are also given in the 
+[repository_aois notebook](dev/repository_aois.ipynb). It is good practice
+to add your notebook to the list of notebooks using each AOI.
+
+If the AOI for a notebook cannot be satisfied by the the AOIs already in use
+in the repository, then email <devrel@planet.com> to request that the demo
+data coverage be expanded to include the new AOI. Once approved, add the AOI to the
```
Comment (sarasafavi) [inline CONTRIBUTING.md:28]: >  then email <devrel@planet.com> to request that the demo data coverage be expanded to include the new AOI.

let's rephrase this as something like _"then email <devrel...> so that we can consider expanding our demo data coverage to include a new AOI."_ I only say this because I don't want to give the wrong impression about the ease of adding new demo data.

Comment (jreiberkyle) [reply CONTRIBUTING.md]: Yes! thank you for catching that. changed.
